app/utils/trayWithText.js

```diff
@@ -0,0 +1,262 @@
+        this.current = this.from
+        return this
+      },
+
+      next () {
+        if (this.current <= this.to) {
+          return { done: false, value: this.current++ }
+        } else {
+          return { done: true }
+        }
+      }
+    }
+
+    for await (const number of range) {
+      if (number < 10) {
+        try {
+          const img = await mergeImg([
+            { src: baseImages + mac + 'iconFreeSpace.png' },
+            { src: baseImages + mac + number + color + '.png' },
+            { src: baseImages + mac + 'iconFreeSpace.png' }
+          ])
+          await img.write(
+            pathToImages + mac + '0' + number + color + '.png',
+            () =>
+              log.debug(
+                'done' + pathToImages + mac + '0' + number + color + '.png'
+              )
+          )
+        } catch (e) {
+          log.debug(e + 'safely ignored error')
+        }
+      } else {
+        try {
+          const lastDigit = number % 10
+          const decimalDigit = Math.floor((number / 10) % 10)
+          const img = await mergeImg([
+            { src: baseImages + mac + decimalDigit + color + '.png' },
+            {
+              src: baseImages + mac + lastDigit + color + '.png'
+            }
+          ])
+          await img.write(pathToImages + mac + number + color + '.png')
+        } catch (e) {
+          log.debug(e + 'safely ignored error')
+        }
+      }
+    }
+  }
+
+  async generateNumbersWithTray (
+    color = 'b',
+    createdName = 'traytMacMonochrome',
+    mac = 'm',
+    iconPath = '/home/m/p/stretchly/app/images/app-icons/traytMacMonochrome.png'
+  ) {
+    const range = {
+      from: 0,
+      to: 99,
+      [Symbol.iterator] () {
+        this.current = this.from
+        return this
+      },
+
+      next () {
+        if (this.current <= this.to) {
+          return { done: false, value: this.current++ }
+        } else {
+          return { done: true }
+        }
+      }
+    }
+
+    for await (const number of range) {
+      const decimalDigit = Math.floor((number / 10) % 10)
+      let leadingZero = ''
+      if (decimalDigit === 0) leadingZero = '0'
+      try {
+        let pictureOffset = -32
+        let macTemplateEnd = ''
+        if (mac === 'm') {
+          pictureOffset = -16
+          macTemplateEnd = 'Template'
+        }
+        const img = await mergeImg([
+          {
+            src: iconPath
+          },
+          {
+            src: pathToImages + mac + leadingZero + number + color + '.png',
+            offsetX: pictureOffset
+          }
+        ])
+        await img.write(
+          pathToImages + createdName + number + macTemplateEnd + '.png'
+        )
+      } catch (e) {
+        log.debug(e + 'safely ignored error')
+      }
+    }
+  }
+
+  async generateCirclesWithTray (
+    color = 'b',
+    createdName = 'traytMacMonochrome',
+    mac = 'm',
+    iconPath = '/home/m/p/stretchly/app/images/app-icons/traytMacMonochrome.png'
```

Comment:
This should be relative path, right?

Reply:
1. no. Images always have to have real path before operations with them. But it could probably be build with use of path lib. As this is methods for generation only and it is default parameter it could be like that, but of course to work for everyone it should be combination of path+ 'app/... '

Reply:
If this is used then I would prefer to have `path` so that we don't need to change it when running it.

Reply:
They are not used and should not be in test coverage. I could safely remove them, I left them just for conviniece of generating new icons version if in future will be needed.